tests/VolleyManagement.UnitTests/Services/GameReportService/Comparers/StandingsEntryComparer.cs

```diff
@@ -10,8 +10,21 @@
     /// Represents a comparer for <see cref="StandingsEntry"/> objects.
     /// </summary>
     [ExcludeFromCodeCoverage]
-    internal class StandingsEntryComparer : IComparer<StandingsEntry>, IComparer
+    public class StandingsEntryComparer : IComparer<StandingsEntry>, IComparer
```

Comment:
Why public?

Reply:
because when I created one constructor, I should make constructor internal or class public, because an instance of this class is a parameter in constructor